spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -222,10 +215,152 @@
           user_id: user.uuid,
         )
 
+        expect(@analytics).to receive(:track_event).with(
+          Analytics::IDV_DOC_AUTH_SUBMITTED_PII_VALIDATION,
+          success: true,
+          errors: {},
+          user_id: user.uuid,
+        )
+
         action
 
         expect_funnel_update_counts(user, 1)
       end
+
+      context 'but doc_pii validation fails' do
+        let(:first_name) { 'FAKEY' }
+        let(:last_name) { 'MCFAKERSON' }
+        let(:state) { 'ND' }
+        let(:dob) { '10/06/1938' }
+
+        before do
+          IdentityDocAuth::Mock::DocAuthMockClient.mock_response!(
+            method: :get_results,
+            response: IdentityDocAuth::Response.new({
+              success: true,
+              errors: {},
+              extra: { result: 'Passed', billed: true },
+              pii_from_doc: {
+                first_name: first_name,
+                last_name: last_name,
+                state: state,
+                dob: dob,
+              },
+            }),
```

Comment:
nit: drop the optional curly braces:

```suggestion
            response: IdentityDocAuth::Response.new(
              success: true,
              errors: {},
              extra: { result: 'Passed', billed: true },
              pii_from_doc: {
                first_name: first_name,
                last_name: last_name,
                state: state,
                dob: dob,
              },
            ),
```